packages/adapters/bitcoin/src/connectors/WalletStandardConnector.ts

```diff
@@ -0,0 +1,67 @@
+import { getWallets } from '@wallet-standard/app'
```

Comment:
Is this required? Thought WalletStandard was a solana thing

Reply:
It was, then it became chain agnostic with https://github.com/wallet-standard/wallet-standard. We are willing to support the extension https://github.com/ExodusMovement/bitcoin-wallet-standard, which is based from wallet standard.